docker/CMSRucioClient/scripts/syncRolesToAttributesLocalUsers.py

```diff
@@ -93,18 +93,56 @@ def set_local_identities(client, site, dns=None, user_map=None, site_map=None):
         client.del_identity(account=account, identity=identity, authtype='X509')
 
 
+def set_group_identities(client, group, dns, dn_email_map):
+
+    account = group
+    email = account + "@cern.ch"
+
+    try:
+        print('Checking for account %s in Rucio' % account)
+        client.get_account(account)
+    except AccountNotFound:
+        print('Adding group account %s with %s' % (account, email))
+        client.add_account(account, 'GROUP', email)
+
+    update_identities(client, account, dns, dn_email_map)
+
+
+def update_identities(client, account, dns, dn_email_map):
```

Comment:
This is a nice refactoring, but you should use it in the place it originally came from too.

Reply:
Yes, I was trying to avoid refactoring existing code in the same commit. But since it is not many lines, I think it's okay to do so. Will do that.